test/account_multisig.test.js

```diff
@@ -0,0 +1,126 @@
+/* global BigInt */
+const nearApi = require('../lib/index');
+const fs = require('fs');
+const BN = require('bn.js');
+const testUtils  = require('./test-utils');
+const semver = require('semver');
+
+let nearjs;
+let startFromVersion;
+
+const {
+    KeyPair,
+    transactions: { functionCall },
+    InMemorySigner,
+    keyStores: { InMemoryKeyStore },
+    multisig: { AccountMultisig, MULTISIG_GAS, MULTISIG_DEPOSIT },
+    utils: { format: { parseNearAmount } }
+} = nearApi;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000;
+
+const AccountMultisigInstance = async (account, keyMapping = ({ public_key: publicKey }) => ({ publicKey, kind: 'phone' })) => {
+    // modifiers to functions replaces contract helper (CH)
+    const { accountId } = account;
+    const keys = await account.getAccessKeys();
+    const accountMultisig = new AccountMultisig(nearjs.connection, accountId, {
+        // skip this (not using CH)
+        getCode: () => {},
+        sendCode: () => {},
+        // auto accept "code"
+        verifyCode: () => ({ success: true, res: '' }),
+        onResult: async () => {
+            const { requestId } = accountMultisig.getRequest();
+            console.log('requestId', requestId);
```

Comment:
if we are going to keep the log in committed code – let's add some specific message. Otherwise best to cleanup debug logs